### ISSUE TITLE
Installation not possible on MAC ARM M1?

### ISSUE THREAD

**Issue by PhilippKIT** [Open-MSS/mss-install#24]
Executing the automatic installation script ./LinuxMac.sh
results in

Checking Mamba installation...
Aborting. Mac ARM M1 currently not supported.

Is there any way to include support for Mac ARM M1 architectures?

**Comment by ReimarBauer** [comment]
good idea. I've no M Processor system. Would it be possible for you to send a pull request?

Since 9.0.0 we have all dependencies availaible for M1-3 processors.
https://anaconda.org/conda-forge/mss

**Comment by ReimarBauer** [comment]
see also https://github.com/Open-MSS/mss-install/issues/25
